theia-extensions/viewer-prototype/src/browser/trace-server-preference.ts

```diff
@@ -15,15 +16,21 @@ export const ServerSchema: PreferenceSchema = {
         },
         [TRACE_PORT]: {
             type: 'number',
-            default: TRACE_SERVER_DEFAULT_PORT,
-            description: 'Specify the port on which you want to execute the server. This change will take effect the next time you open a trace',
+            default: TRACE_VIEWER_DEFAULT_PORT,
+            description: 'Specify the port the view is using to connect to the trace server',
```

Comment:
I suggest:
`Specify the port the Trace Viewer would use to connect to the trace server'